src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -675,6 +719,7 @@ def load_pvt_models_description(
         "tables": load_pvt_tables,
         "correlations": load_pvt_model_correlation_description,
         "compositions": load_pvt_model_compositional_description,
+        "combined": load_pvt_model_combined_description,
```

Comment:
Could you try to partially use the auto loaders?
And remove `load_pvt_model_combined_description` and `load_combined_fluid_description`?

```suggestion
        "combined": get_instance_loader(class_=case_description.PvtModelCombinedDescription),
```